miden/tests/integration/operations/ext2_ops.rs

```diff
@@ -0,0 +1,125 @@
+use crate::build_op_test;
+use rand_utils::rand_value;
+use vm_core::{Felt, FieldElement, QuadExtension, StarkField};
+
+type ExtElement = QuadExtension<Felt>;
+
+// EXT2 OPS ASSERTIONS - MANUAL TESTS
+// ================================================================================================
+
+#[test]
+fn ext2add() {
+    let asm_op = "ext2add";
+
+    let a_ext = rand_value::<ExtElement>();
+    let b_ext = rand_value::<ExtElement>();
+    let a_ext_arr = [a_ext];
+    let b_ext_arr = [b_ext];
+    let c_ext_arr = vec![a_ext + b_ext];
+    let a = ExtElement::as_base_elements(&a_ext_arr);
+    let b = ExtElement::as_base_elements(&b_ext_arr);
+    let c = ExtElement::as_base_elements(&c_ext_arr);
```

Comment:
nit: since we do these conversions quite a bit, it might be better to create a helper functions. Something like:
```Rust
fn ext_element_to_ints(a: ExtElement) -> (u64, u64)
```